public/locales/ar/translation.json

```diff
@@ -8,6 +8,309 @@
     "buttons": {
       "signIn": "تسجيل الدخول",
       "signUp": "الاشتراك"
+    },
+    "userSettings": {
+      "userIcon": "صورة المستخدم",
+      "profile": "صفحتك الرئيسية",
+      "signOut": "تسجيل الخروج"
+    }
+  },
+  "home": {
+    "seekHelpSection": [
+      {
+        "title": "الدكتورة",
+        "name": "جيني ويلسون",
+        "portraitImg": "doctorImg",
+        "phoneNumber": "+1-340-5558-235",
+        "address": "٩٠٩٤ شارع باي ميادوز ، مدينة كونري ، ٣٠٠١٢",
+        "moreInfo": "لوريم ايبسوم دولار سيت أميت ,كونسيكتيتور أدايبا يسكينج أليايت,سيت دو أيوسمود تيمبور أنكايديديونتيوت لابوري ات دولار ماجنا أليكيوا . يوت انيم أد مينيم"
+      },
+      {
+        "title": "المحامية",
+        "name": "جيني ويلسون",
+        "phoneNumber": "+1-340-5558-235 ",
+        "address": "٩٠٩٤ شارع باي ميادوز ، مدينة كونري ، ٣٠٠١٢",
+        "moreInfo": "لوريم ايبسوم دولار سيت أميت ,كونسيكتيتور أدايبا يسكينج أليايت,سيت دو أيوسمود تيمبور أنكايديديونتيوت لابوري ات دولار ماجنا أليكيوا . يوت انيم أد مينيم"
+      },
+      {
+        "title": "الدكتورة النفسانية",
+        "name": "جيني ويلسون",
+        "portraitImg": "doctorImg",
+        "phoneNumber": "+1-340-5558-235 ",
```

Comment:
i'm wondering, if you don't provide this translation, does it fall back to the english one? if that's case you should just be deleting these phone numbers, leaving them "empty" in non-english translations. same with portraitImg.

Reply:
we dont want to write the data duplicated

Reply:
never mind, i dont think this is possible because its stored in an array like this (?)

Reply:
for now it's better to keep the number in english because in arabic we write the number from rtl so the number is coming wrong. if we manage to flip the website on arabic we can make them arabic.